### PR TITLE
Okta SSO guide updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updated steps to set up Okta SSO using the RudderStack Okta integration on OIN.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/user-guides/administrators-guide/okta-sso/index.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+2. Set the **Application Label** that you set previously. Then, click **Next**.
+
+<div class="warningBlock">
+Do not check the <strong>Do not display application icon to users</strong> and <strong>Do not display application icon in the Okta Mobile App</strong> settings as this app will be visible to your users.
+</div>
+
+<img src="../../../assets/user-guides/rudderstack-okta-sso-5.png" alt="Application name and visibility" />
+
+3. Under **Sign on methods**, choose **Bookmark-only**. Set the **Login URL** to `https://app.rudderstack.com/sso?domain=<your_website>.com`, where `<your_website>` is your organizaiton's web domain. Retain the rest of the settings and click **Done**.
```
Comment (aris1009) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:69]: is it possible to have a "click to copy" button for the URL?

**Review by aris1009 — APPROVED** [review]
LGTM

File: docs/user-guides/administrators-guide/okta-sso/index.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:69]:
```suggestion
3. Under **Sign on methods**, choose **Bookmark-only**. Set the **Login URL** to `https://app.rudderstack.com/sso?domain=<your_website>.com`, where `<your_website>` is your organization's web domain. Retain the rest of the settings and click **Done**. 
```

Comment (ameypv-rudder) [reply docs/user-guides/administrators-guide/okta-sso/index.mdx:69]: As discussed, we'll assess if we can add this functionality to our new Hugo site.

File: docs/user-guides/administrators-guide/okta-sso/index.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+The <a href="https://www.okta.com/integrations/rudderstack/">Okta RudderStack app</a> is available on the **Okta Integration Network (OIN)**. This guide lists the steps to set up the integration.
+
+<GhBadge
+  url={'https://rudderstack.com/enterprise-quote'}
+  label={'Plan'}
+  message={'Enterprise'}
+  color={'blueviolet'}
+/>
+
+<div class="infoBlock">
```
Comment (ishachopra0401) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:16]: Can remove the infoblock as we are following in other docs as well.

File: docs/user-guides/administrators-guide/okta-sso/index.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+3. In the **Configuration Steps** section of the guide, copy the **Identity Provider Single Sign-On URL** present in **Step 2**, as shown:
+
+<img src="../../../assets/user-guides/rudderstack-okta-sso-4.png" alt="Single Sign-On URL" />
+
+4. Share this URL with the [RudderStack team](mailto:hello@rudderstack.com) to enable SAML 2.0 for your account.
+
+## Step 3: Add the RudderStack SSO Bookmark app
+
+<div class="infoBlock">
+Your users will use this app to quickly access the RudderStack web app using the SSO functionality.
```
Comment (ishachopra0401) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:54]:
```suggestion
Your users can use this app to quickly access the [RudderStack dashboard](https://app.rudderstack.com/) using the SSO functionality.
```

File: docs/user-guides/administrators-guide/okta-sso/manual-setup.mdx [hunk 4]
```diff
@@ -1,13 +1,9 @@
 ---
-title: "Okta SSO Setup"
-description: Set up the RudderStack SSO (Single Sign-On) feature with Okta.
+title: "Okta SSO Manual Setup"
```
Comment (ishachopra0401) [inline docs/user-guides/administrators-guide/okta-sso/manual-setup.mdx:2]: Assuming that we will add this to the sidebar as well.

**Review by ishachopra0401 — APPROVED** [review]

File: docs/user-guides/administrators-guide/okta-sso/manual-setup.mdx (hunk 4 above)
Comment (ameypv-rudder) [reply docs/user-guides/administrators-guide/okta-sso/manual-setup.mdx:2]: No we want to hide this in the sidebar, but we can retain this title.

File: docs/user-guides/administrators-guide/okta-sso/index.mdx (hunk 2 above)
Comment (ameypv-rudder) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:16]:
```suggestion
```

File: docs/user-guides/administrators-guide/okta-sso/index.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+<GhBadge
+  url={'https://rudderstack.com/enterprise-quote'}
+  label={'Plan'}
+  message={'Enterprise'}
+  color={'blueviolet'}
+/>
+
+<div class="infoBlock">
+To manually configure and enable Okta SSO for your organization, refer to the <Link to="/user-guides/administrators-guide/okta-sso/manual-setup/">Manual Setup</Link> guide. 
+</div>
```
Comment (ameypv-rudder) [inline docs/user-guides/administrators-guide/okta-sso/index.mdx:18]:
```suggestion
```

**Pull request merged**